### ISSUE TITLE
Compiled DDM Integrator function result is off by a factor of 20

### ISSUE THREAD

**Issue by SamKG** [PrincetonUniversity/PsyNeuLink#1509]
The following code:

```Python
import psyneulink as pnl
import numpy as np
import pytest

decisionMaker = pnl.DDM(function=pnl.DriftDiffusionIntegrator(starting_point=0,
                                                              threshold=1,
                                                              noise=0.0),
                        reinitialize_when=pnl.AtTrialStart(),
                        output_ports=[pnl.DECISION_VARIABLE, pnl.RESPONSE_TIME],
                        name='DDM')

comp = pnl.Composition()
comp.add_node(decisionMaker)

print(comp.run([.05], termination_processing={pnl.TimeScale.TRIAL: pnl.WhenFinished(decisionMaker)},
         bin_execute=False))
```

results in an output of 

> [array([[1.0]], dtype=object), array([[20.0]], dtype=object)]

in Python mode, but

> [[0.05], [1.0]]

in compiled mode - off by a factor of exactly 20.

**Comment by jvesely** [comment]
This looks like a problem with 'execute_until_finished'. iirc, `RESPONSE_TIME` refers to number of iterations executed.
setting termination processing in the `run` method is also not going to work with compilation.

**Comment by SamKG** [comment]
Fixed in #1555.
